foca/api/register_openapi.py

```diff
@@ -5,7 +5,7 @@
 from typing import Dict, List
 
 from connexion import App
-import yaml
+# import yaml
```

Comment:
Please just delete any lines you don't need anymore. Everything is version controlled, so there is no need to keep unused code - if we need to revert or reuse unused code later on, we can always go back.